src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -113,6 +114,8 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      */
     void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+        companyListPanel = new CompanyListPanel(logic.getFilteredCompanyList());
+        eventListPanel = new EventListPanel(logic.getFilteredEventList());
         entryListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
```

Comment:
Line 117 and 118 should not be there I think. This is because the list will display the personList. In that case, line 117 and 118 will not do anything as they are just a placeholder to hold the companyList and the eventList

Reply:
Line 117 and 118 are there because we only need to create the CompanyListPanel and EventListPanel objects once. The reason for this is in the conversation below.